### PR TITLE
Migrate unitSummary amiPercentage column to int

### PR DESCRIPTION
**Pull request by avaleske**
# Pull Request Template

## Issue

Addresses #478

- [x] This change addresses only certain aspects of the issue

## Description
Migrate the amiPercentage column on the unit summary table to an integer, so we can filter based on it.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Can This Be Tested/Reviewed?

Ran the tests and built the site

- [ ] Desktop View
- [ ] Mobile View
- [ ] Test A
- [ ] Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: backend/core/src/units-summary/entities/units-summary.entity.ts [hunk 1]
```diff
@@ -49,11 +49,11 @@ class UnitsSummary {
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   monthlyRentAsPercentOfIncome?: string | null
 
-  @Column({ nullable: true, type: "text" })
+  @Column({ nullable: true, type: "integer" })
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
```
Comment (anders-schneider) [inline backend/core/src/units-summary/entities/units-summary.entity.ts:55]: nit: IsNumber

**Review by anders-schneider — APPROVED** [review]
Hopefully we should be able to contribute this upstream too? (I don't think they're populating the units_summary table yet, so the migration shouldn't be disruptive)

File: backend/core/src/units-summary/entities/units-summary.entity.ts (hunk 1 above)
Comment (avaleske) [reply backend/core/src/units-summary/entities/units-summary.entity.ts]: ah thanks!

**Comment by avaleske** [comment]
Contributing upstream in https://github.com/bloom-housing/bloom/pull/1797

**Pull request merged**